src/governance.rst

```diff
@@ -144,6 +144,25 @@ the time when a vote is called, but may never be less than 24 hrs.
     * Sensitive
     * 75% Majority to pass
 
+:Overall workflow and packaging policies: The proposer can choose to
+    create a poll with an external tool or call
+    for voting on the GH issue in question.
+    The voting must period be open for at least one core
+    member meeting cycle to allow for clarification questions
+    and discussions.
+
+    * Standard
+    * 50% Majority to pass
+
+:Modifying the governance document: The voting should happen in the PR
+    in question and there mus tbe a call to `@conda-forge/core`.
+    The voting period must be open for at least one core
+    member meeting cycle to allow for clarification questions
+    and discussions.
+
+    * Sensitive
```

Comment:
Sensitive votes are private, should this be standard?

Reply:
You are right, maybe sensitive is not the best category here... Let's use Standard with 75%, is that OK?